packages/chrome/src/styled/header/StyledBaseHeaderItem.ts

```diff
@@ -15,17 +15,11 @@ import {
 const COMPONENT_ID = 'chrome.base_header_item';
 
 export interface IStyledBaseHeaderItemProps {
-  /**
-   * Horizontally maximize a flex item in the header to take as much space as possible (i.e. breadcrumb container)
-   **/
+  /** Maximizes the width of a flex item in the header (i.e. breadcrumb container) */
```

Comment:
We should remove ~(i.e. breadcrumb container)~ as design research has determined planting a breadcrumb in the header is off-pattern.